### PR TITLE
CLXD-172

### PR DESCRIPTION
**Pull request by toddcd**
# JIRA

[CLXD-1724](https://clxmedia.atlassian.net/browse/CLXD-1724)

# Description
_This is ready for PR but on-hold for merging into QA branch until QA team resources have freed up._

- fix for archiving projects and it's task's
- removed person company_type from odoo_entity bulk sync

### CODE REVIEW

File: clx_utils/models/clx_mysql.py [hunk 1]
```diff
@@ -139,9 +139,7 @@ def update_odoo_entity_table(self):
         count = 0
         db_config = self._read_db_config()
 
-        entity_records = self.env["res.partner"].search(
-            [("company_type", "in", ("management", "company", "person", "owner"))]
-        )
+        entity_records = self.env["res.partner"].search([("company_type", "in", ("management", "company", "owner"))])
```
Comment (Vlad-CL) [inline clx_utils/models/clx_mysql.py:142]: Just suggestion, maybe this command need to be sudo-ed.

File: clx_task_management/models/project.py [hunk 2]
```diff
@@ -77,11 +77,13 @@ def create(self, vals):
...
     def write(self, vals):
-        res = super(ProjectProject, self).write(vals)
-
         if "active" in vals:
-            for task in self.task_ids:
-                task.write({"active": vals.get("active", False)})
+            isActive = vals["active"]
+            vals.pop("active")
+            self._cr.execute("UPDATE project_project SET active = %s WHERE id = %s", [isActive, self.id])
+            self._cr.execute("UPDATE project_task SET active = %s WHERE project_id = %s", [isActive, self.id])
```
Comment (Vlad-CL) [inline clx_task_management/models/project.py:84]: I hope not using ORM is ok for Odoo flows

**Review by Vlad-CL — APPROVED** [review]

File: clx_utils/models/clx_mysql.py (hunk 1 above)
Comment (toddcd) [reply clx_utils/models/clx_mysql.py:142]: this is called from a cron job

File: clx_task_management/models/project.py (hunk 2 above)
Comment (toddcd) [reply clx_task_management/models/project.py:84]: Me too! Had to do it this was to get around the inheritance issues and a nasty infinite loop with the write function. Testing shows it to work well. 👍

**Pull request merged**